### ISSUE TITLE
_sendRequest Returned SessionID

### ISSUE THREAD

**Issue by arshanskiyav** [zbchristian/huawei_hilink_api#1]
Hi

I want to get sms or ussd, but script return SessionID, calling _getToken in _sendRequest overwrite $response

**Comment by zbchristian** [comment]
Yes, you are correct. This will be the case, if no token is available yet, or all tokens have been used. 

Did you call `_initHilinkAPI` first? This should fill the token list and `_getToken` is not filling `$response`. 

Will have a look and fix this.
